### ISSUE TITLE
Provide 1.7.10 Download

### ISSUE THREAD

**Issue by Jocroy** [MinimallyCorrect/TickProfiler#110]
Since the jenkins page we used to download it until now is down, would it be possible to get a compiled download for 1.7.10? There is a branch here that looks semi-active still. There is at least some need for this in the Greg Tech New Horizons Community.

**Comment by b0bst3r** [comment]
Yeah same with 1.12 if it can be uploaded to Curse as Jenkins is broken again
